### DISCUSSION TITLE
Add delete by date option for manual cleanup.

### DISCUSSION THREAD

**Discussion by jshall77** [aaronwmorris/indi-allsky#1303] [Ideas]
Is it possible to add a function that would allow deleting images/startrails/pano/vids/keograms etc. by date? There was an option like this a while back, but something cleaner in the "Tools" section for this would be nice.

**Comment by aaronwmorris** [comment]
The ability to delete assets is within the Generate view under the Timelapse view.

Merged #1305 with a direct link to Generate under tools.
